public/app/features/alerting/unified/api/onCallApi.ts

```diff
@@ -1,36 +1,79 @@
-import { lastValueFrom } from 'rxjs';
-
-import { getBackendSrv } from '@grafana/runtime';
+import { GRAFANA_ONCALL_INTEGRATION_TYPE } from '../components/receivers/grafanaAppReceivers/onCall/onCall';
+import { SupportedPlugin } from '../types/pluginBridges';
 
 import { alertingApi } from './alertingApi';
-export interface OnCallIntegration {
+
+export interface NewOnCallIntegrationDTO {
+  id: string;
+  connected_escalations_chains_count: number;
+  integration: string;
   integration_url: string;
+  verbal_name: string;
+}
+
+export interface OnCallPaginatedResult<T> {
+  results: T[];
```

Comment:
Currently, we use only one paginated endpoint from the OnCall API, but the interface there is generic and if we need to use another one in the future it will be easier to remember that we already have it in place